src/components/common/animations/Feed/Base.js

```diff
@@ -10,11 +11,21 @@ class FeedInfo extends React.Component {
   }
 
   componentDidMount() {
-    const { delay = 0 } = this.props
+    const { delay = 0, asImage } = this.props
 
     this.anim.onComplete = this.onAnimationFinishHandler
```

Comment:
why not just onComplete={this.onAnimationFinishHandler} in jsx ?

Reply:
There is no onComplete prop described in docs. There is 'onAnimationFinish' prop but it does not work on the web. So we found onComplete from reference.

``` this.anim.onComplete = this.onAnimationFinishHandler ```

This implementation works on the web, and not working for native apps.